less/plugins/adapt-contrib-slider/slider.less

```diff
@@ -67,11 +70,12 @@
   }
 
   &__fill {
-    background-color: @item-color;
+    background-color: @greyDark;
```

Comment:
A hard coded variable here would cause the slider fill to never be editable in the Authoring Tool. How does the slider fill look when both the handle and fill are set to use `@item-color`?